dockerFiles/marklogic-server-centos:base

```diff
@@ -31,6 +32,12 @@ RUN yum localinstall -y /tmp/marklogic-server.rpm \
     && yum -y install sudo \
     && yum -y clean all
 
+
+
+###############################################################
+# Copy converters package 
+###############################################################
+COPY ${ML_CONVERTERS} /opt/converters.rpm
```

Comment:
@jkerr5 @vitalykorolev Not sure if there is a preferred place that we want to put the converters RPM to live in the docker image. I just chose the `/opt/` folder without much thought.